winui/Rating/Overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: About WinUI Rating control| Syncfusion
+description: Learn here all about introduction of Syncfusion WinUI Rating Control (SfRating) with key features and more.
+platform: WinUI
+control: SfRating
+documentation: ug
+---
+
+# WinUI Rating Overview
+
+The Syncfusion provides the number of stars that represents a rating. And also used to configure the item size, tooltip and the number of displayed items in the SfRating control. The Rating control can be used in various scenarios like movie rating, rating the application and etc.
```

Comment:
@Priyanka-Vijayasankar Change the overview and key features content with the following PR. https://github.com/syncfusion-content/website-whatsnew/pull/367/files

Reply:
Changed the overview and key features content above mentioned PR.